Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    datadog-notifications (0.5.3)
+    datadog-notifications (0.6.0)
```

Comment:
it's not a major/breaking change, nothing's that using this lib needs to adapt anything, it's just an additional tag, you are not removing anything or changing the API. 0.5.4 pls